test/check-application

```diff
@@ -2065,6 +2069,110 @@ class TestFiles(testlib.MachineCase):
         b.wait_in_text("h4.pf-v5-c-alert__title", "Pasting failed")
         b.wait_in_text(".pf-v5-c-alert__description", "\"newfile\" exists")
         self.assertEqual(m.execute("head -n 1 /home/admin/kbdCopy/newfile"), "keybindings good\n")
+        b.click(".pf-v5-c-alert__action button")
+
+        # Copy & paste as superuser
+        # Switch to list so owner is visible
+        b.click("button[aria-label='Display as a list']")
+        b.go("/files#/?path=/home/admin")
+        self.assert_last_breadcrumb("admin")
+        b.go("files#/?path=/home/admin")
+        self.assert_last_breadcrumb("admin")
+        m.execute("useradd -m foouser")
+
+        b.click("[data-item='newfile']")
+        b.click("#dropdown-menu")
+        b.click("#copy-item")
+
+        b.go("files#/?path=/home/foouser")
+        self.assert_last_breadcrumb("foouser")
+        # Paste file as destination directory owner
+        b.click("#dropdown-menu")
+        b.click("#paste-item")
+        b.wait_visible("#paste-owner-modal")
+        b.click("#paste-owner-modal button.pf-m-primary")
+        b.wait_in_text("[data-item='newfile'] .item-owner", "foouser")
```

Comment:
... and also ensure that the copied file has the expected content and permissions.